content/en/docs/setup/windows/intro-windows-in-kubernetes.md

```diff
@@ -0,0 +1,400 @@
+
+{{% thirdparty-content %}}
+
+#### cri-containerd
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+You can use {{< glossary_tooltip term_id="containerd" text="ContainerD" >}} 1.4.0+
+as the container runtime for Kubernetes nodes that run Windows.
+
+Learn how to [install ContainerD on a Windows node](/docs/setup/production-environment/container-runtimes/#install-containerd).
+
+{{< note >}}
+There is a [known limitation](/docs/tasks/configure-pod-container/configure-gmsa/#gmsa-limitations)
+when using GMSA with containerd to access Windows network shares, which requires a
+kernel patch.
+{{< /note >}}
+
+#### Mirantis Container Runtime {#mcr}
+
+[Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR) is available as a container runtime for all Windows Server 2019 and later versions.
+
+See [Install MCR on Windows Servers](https://docs.mirantis.com/mcr/20.10/install/mcr-windows.html) for more information.
+
+## Windows OS version compatibility {#windows-os-version-support}
+
+On Windows nodes, strict compatibility rules apply where the host OS version must
+match the container base image OS version. Only Windows containers with a container
+operating system of Windows Server 2019 are fully supported.
+
+For Kubernetes v{{< skew currentVersion >}}, operating system compatibility for Windows nodes (and Pods)
+is as follows:
+
+Windows Server LTSC release
+: Windows Server 2019
+: Windows Server 2022
+
+Windows Server SAC release
+:  Windows Server version 20H2
+
+The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
+
+## Getting help and troubleshooting {#troubleshooting}
+
+Your main source of help for troubleshooting your Kubernetes cluster should start
+with the [Troubleshooting](/docs/tasks/debug-application-cluster/troubleshooting/)
+page.
+
+Some additional, Windows-specific troubleshooting help is referenced here. Logs are an 
+important element of troubleshooting issues in Kubernetes. Make sure to include them 
+any time you seek troubleshooting assistance from other contributors. Follow the 
+instructions in the SIG Windows 
+[contributing guide on gathering logs](https://github.com/kubernetes/community/blob/master/sig-windows/CONTRIBUTING.md#gathering-logs).
+
+* [Troubleshooting Windows Networking](/docs/setup/windows/windows-networking#troubleshooting-network)
+* [Node-level Troubleshooting](/docs/setup/windows/user-guide-windows-containers#troubleshooting-node)
+
+### Further investigation
+
+If these steps don't resolve your problem, you can get help running Windows containers on Windows nodes in Kubernetes through:
+
+* StackOverflow [Windows Server Container](https://stackoverflow.com/questions/tagged/windows-server-container) topic
+* Kubernetes Official Forum [discuss.kubernetes.io](https://discuss.kubernetes.io/)
+* Kubernetes Slack [#SIG-Windows Channel](https://kubernetes.slack.com/messages/sig-windows)
```

Comment:
Let's omit this from the overview page - we should instead add that to all of the Windows-specific troubleshooting pages.

(As a later piece of work, anyone who's interested in cleanup can refactor it into a reusable shortcode).